iap/README.md

```diff
@@ -76,6 +76,26 @@ service account private key can impersonate that account!
    ```
 2. Copy `validate_jwt.py` into your application.
 
+## Using generate_self_signed_jwt
+
+### Self-signed JWT with IAM Credentials API
+
+1. Install the libraries listed in `requirements.txt`, e.g. by running:
+   ```
+   virtualenv/bin/pip install -r requirements.txt
+   ```
+2. Call sign_jwt with service account email and target resource url.
+3. Use resulting token to access application.
```

Comment:
What is "application"? If this isn't self-explanatory, please provide a bit more explanation.